api/task.go

```diff
@@ -219,6 +219,7 @@ func (h TaskHandler) Update(ctx *gin.Context) {
 	db := h.DB.Model(m)
 	db = db.Where("id", id)
 	db = db.Where("state", tasking.Created)
+	db = db.Omit("Bucket")
```

Comment:
The read-only permission removed from the model so we need to prevent updating the bucket ref in the API.